src/include/planner/index_scan_plan.h

```diff
@@ -56,8 +56,7 @@ class IndexScanPlan : public AbstractScan {
      * be called to notify later procedures of the absense of an index
      */
     IndexScanDesc(
-        oid_t p_index_id,
-        const std::vector<oid_t> &p_tuple_column_id_list,
+        oid_t p_index_id, const std::vector<oid_t> &p_tuple_column_id_list,
```

Comment:
Is putting both declarations on the same line intentional? I didn't see anything in our coding style.
I would have expected that we prefer separate lines.

Reply:
The clang-format on my machine just automatically adjusted it that way... I run on-save formatting automatically. I've adjusted the format manually.